website/docs/configuration/modules.html.md

```diff
@@ -372,6 +372,8 @@ resource "aws_s3_bucket" "example" {
 We recommend using this approach when a single configuration for each provider
 is sufficient for an entire configuration.
 
+~> **Note:** Only provider configurations are inherited by child modules, not provider source or version requirements. Each module must [declare its own provider requirements](provider-requirements.html). This is especially important for non-HashiCorp providers.
```

Comment:
Other local links have a leading `./`, though I can't actually promise that it's required: 
`(./provider-requirements.html)`

Reply:
Huh! I just checked and both work, and we use both even within this file. I'm not sure why the inconsistency is there…